src/components/BannerComponent.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import {useState,useEffect} from 'react'
+import '../Styles/featureBanner.css'
+
+export const Banner = ({slide}) => {
+    const [count,setCount]=useState(0);
```

Comment:
A format style is always appreciated in a project for clean code conventions :), I like to use prettier or simpe eslint rules